selfdrive/locationd/torqued.py

```diff
@@ -16,7 +16,9 @@
 HISTORY = 5  # secs
 POINTS_PER_BUCKET = 1500
 MIN_POINTS_TOTAL = 4000
+MIN_POINTS_TOTAL_QLOG = 800
```

Comment:
is this a factor derived from the decimation? if so, should use the decimation from services.py

Reply:
Not really, it was tuned to be good enough for an approximate estimate of the values.